src/coreclr/tools/aot/ILLink.Shared/TrimAnalysis/HandleCallAction.cs

```diff
@@ -773,7 +773,7 @@ GenericParameterValue genericParam
 
 					foreach (var typeNameValue in argumentValues[0]) {
 						if (typeNameValue is KnownStringValue knownStringValue) {
-							if (!_requireDynamicallyAccessedMembersAction.TryResolveTypeNameAndMark (knownStringValue.Contents, false, out TypeProxy foundType)) {
```

Comment:
Anything in the ILLink.Shared project should be left as-is (and warning suppressed, or even better suppressed in editorconfig for the entire project). This code is shared verbatim with the linker where the parameters are used.

Reply:
reverted